src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
+        Random random = new Random();

```

Comment:
same, lets make Random as class level variable, not to create new instance of Random class every time we call getRandomBall() method